package-lock.json

```diff
@@ -3828,4 +3828,4 @@
       "dev": true
     }
   }
-}
```

Comment:
Can you remove this change?

Reply:
Let me explain. 

First, the `package-lock.json` autogenerated and I don't know why it was different with your file. I commit it and pushed it here. Then you requested the change and I copied **yours** file and put it back. 

Now if you request to remove it, I will do it. 

But I'm not sure about the future.